docs/metecho-how-to.md

```diff
@@ -0,0 +1,150 @@
+# Steps to Guide a Team Through Metecho
+
+## Step 0 - Log in with GitHub
+- Create a GitHub account, if you don’t yet have one.
+- Contact an admin for the repository (e.g. Nonprofit Success Pack) on GitHub and ensure that you and anyone you’d like to work with are collaborators. 
+
+If you want to create a project, skip down to [Create a Project](#create-a-project).
+
+
+## Step 1 - Select a Project
+**Project** - *(e.g. Nonprofit Success Pack) Projects contain all the work being done. They are equivalent to GitHub repositories.*
+
+- Select the Project you want to contribute to. 
+- If you do not see the Project you want to work on: 
+  - Confirm that you are logged into the correct Metecho account.
+  - Contact an admin for the repository on GitHub and ensure that you are a collaborator. 
+  - Select Re-sync Projects in Metecho.
+
+To continue, go to **Step 2 - Create a Task/s** or, if you have a set of several closely related Tasks in mind, skip to [Step 13 - Create an Epic/s](#step-13---create-an-epics)
+
+## Step 2 - Create a Task/s
+**Task** - *Tasks represent small changes to the Project, and may stand alone or be part of an Epic. Creating a Task creates a branch in GitHub.*
+
+- Select the Tasks tab. 
+- Create a new Task of your own (or from an existing GitHub Issue).
+  - Name the Task. 
+- The name can be a brief description of the work.
+- Use the optional description section for more detail, if needed.
+- Use the recommended dev org type unless you know of a specific reason to use a different one.
+- Select Create to save and navigate to your Task detail view. Select Create & New to continue creating multiple Tasks that are unrelated to each other. 
+- If you need to edit the Task name or description, or delete the Task, select the gear icon in the top right corner. Deleting a Task deletes all the Orgs as well.
+
+## Step 3 - Invite Task Developer/s  
+**Developer** - *The person invited to do the work of a Task.* 
+
+- Assign yourself or invite another person to be the Developer for each Task you created. 
+- Invited Developers will see an alert in Metecho and receive an email invitation. They need to accept the invitation if they want to begin development on the Task.
```

Comment:
This might not be entirely correct since we don't have the invitation system in place yet